core/arch/arm/plat-imx/drivers/tzc380.c

```diff
@@ -50,5 +50,7 @@ void imx_configure_tzasc(void)
 		region = tzc_auto_configure(CFG_SHMEM_START, CFG_SHMEM_SIZE,
 			     TZC_ATTR_SP_ALL, region);
 		DMSG("Action register: %xl", tzc_get_action());
+		if (tzc_regions_lockdown() < 0)
```

Comment:
minor: should use `if (tzc_regions_lockdown())` or `if (tzc_regions_lockdown() != TEE_SUCCESS)` here rahter than in next commit.